Model/Transaction.php

```diff
@@ -106,6 +107,20 @@ public function getChildTransactions()
         return $this->_aChildTransactions;
     }
 
+    /**
+     * Returns true is transaction's payment method is Payment on Invoice or Payment in Advance
```

Comment:
there is a typo, Returns true "is", i think its probably "if".

Reply:
👍